torch/csrc/jit/tensorexpr/external_functions.cpp

```diff
@@ -225,6 +225,24 @@ void nnc_aten_addmm(
   }
 }
 
+void nnc_aten_digamma(
```

Comment:
It would be good to add a test for this too, like we have for other external calls (in https://github.com/pytorch/pytorch/blob/master/test/cpp/tensorexpr/test_external_calls.cpp)